src/lib/components/Table.svelte

```diff
@@ -43,71 +63,115 @@
     }
 </script>
 
-<table class="table table-pin-rows">
-    <thead>
-        <tr class="bg-base-200">
-            {#if enableSelectAll}
-                <th>
-                    <input
-                        type="checkbox"
-                        class="checkbox checkbox-sm"
-                        on:click={onSelectAll}
-                        checked={allItemsSelected}
-                        disabled={items.length === 0}
-                    />
-                </th>
-            {/if}
-            {#each columns as { text, sortKey, itemKey, width } (itemKey)}
-                {#if sortKey}
-                    <SortingTableHeaderCell
-                        {text}
-                        {sortKey}
-                        bind:currentSort={searchParams.sort}
-                        style={width ? `width: ${width}ch;` : ''}
-                    />
-                {:else}
-                    <th style={width ? `width: ${width}ch;` : ''}>{text}</th>
-                {/if}
-            {/each}
-        </tr>
-    </thead>
-    <tbody>
-        {#each items as item, index (idColumn === 'index' ? index : item[idColumn])}
-            {@const href = itemUrlPrefix && `${itemUrlPrefix}${idColumn === 'index' ? '' : item[idColumn] ?? ''}`}
-            <tr class="hover">
-                {#if enableSelectAll || enableSelect}
-                    <TableCell class="w-4">
+<div bind:this={scrollingDiv} class="max-h-full flex-1 overflow-y-scroll {$$props.class}">
```

Comment:
This div was previously handled by whatever component was using the Table. It was nearly always the same, and moving here will ensure the pagination component renders at the bottom of the page like it should.